app/res/xml/preferences_developer.xml

```diff
@@ -173,4 +173,9 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-certificate-transparency"
         android:title="Certificate Transparency"/>
+    <EditTextPreference
+        android:defaultValue="@string/logs_server_url_protocol"
+        android:enabled="true"
+        android:key="log_receiver_url"
+        android:title="Log Submission Server"/>
```

Comment:
As per the discussion  below, I think we should rename the text on this to be something like "Override: Custom Log Submission Server" so that it's clear that this is a secondary setting, not the baseline.

Reply:
Done 9e623dfe3579c385de1f1efe58a49dc5cbb95cb7